### PR TITLE
[fleet] add multi cluster update commands

### PR DESCRIPTION
**Pull request by pdaru**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Description
- add support for multi cluster update commands
- add completers & validators for new commands
- add help text for new commands

### Related command
- az fleet updaterun create 
- az fleet updaterun update
- az fleet updaterun delete
- az fleet updaterun show
- az fleet updaterun list
- az fleet updaterun start
- az fleet updaterun stop

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
fleet

**Comment by kairu-ms** [comment]
Please fix CI issue

**Comment by serbrech** [comment]
LGTM

File: src/fleet/azext_fleet/_params.py [hunk 1]
```diff
@@ -41,3 +42,28 @@ def load_arguments(self, _):
...
+        c.argument('update_group')
+
+    with self.argument_context('fleet updaterun') as c:
+        c.argument('name', options_list=['--name', '-n'], help='Specify name for the update run.')
+        c.argument('fleet_name', options_list=['--fleet-name', '-f'], help='Specify the fleet name.')
+
+    with self.argument_context('fleet updaterun create') as c:
+        c.argument('upgrade_type', validator=validate_upgrade_type)
+        c.argument('kubernetes_version', validator=validate_kubernetes_version)
+        c.argument('stages', type=file_type, completer=FilesCompleter())
```
Comment (kairu-ms) [inline src/fleet/azext_fleet/_params.py:69]: If you use codegen, we can support json, yaml files as well as shorthand sytax for stages property.

Comment (pdaru) [reply src/fleet/azext_fleet/_params.py:69]: ok, we have more incoming changes. can I give that a try then?

Comment (serbrech) [reply src/fleet/azext_fleet/_params.py:69]: can we do this in other iterations?
especially if your codegen doesn't support typespec generated swagger yet?

**Review by kairu-ms — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**